src/ppx_deriving.cppo.ml

```diff
@@ -12,6 +12,21 @@ open Parsetree
 open Ast_helper
 open Ast_convenience
 
+#if OCAML_VERSION >= (4, 05, 0)
+module Typ = struct
+  include Typ
+
+  let poly ?(loc= !Ast_helper.default_loc) ?attrs vars ty =
+    let vars = List.map (fun txt -> { Asttypes.loc; txt }) vars in
+    Typ.poly ~loc ?attrs vars ty
```

Comment:
I think that I added this function recently to deal with polymorphism, and my gut feeling (but I'm not the maintainer) is that it should be ok to change the interface (to mirror a change in the parsetree format that is a net improvement). I don't think it is necessary to do this improvement in the current PR.